libobjrenderer/src/objectsscene.cpp

```diff
@@ -854,6 +854,14 @@ QGraphicsView *ObjectsScene::getActiveViewport(void)
 	return(view_p);
 }
 
+void ObjectsScene::adjustViewportToItems(QList<BaseObjectView *> items)
```

Comment:
What's the purpose of this method? I don't get it.

Reply:
Check https://doc.qt.io/qt-5/qgraphicsview.html#fitInView
It both adjusts zoom and center, I figured it is nice to look at the query items.